softlayer/vm/interface.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 type VMCloudProperties struct {
+	VmNamePrefix             string                               `json:"vmNamePrefix,omitempty"`
```

Comment:
Great, thanks Kara, if possible, I can co-work with you to fix the integration test cases.